src/media-query-provider.js

```diff
@@ -31,10 +33,13 @@ class MediaQueryProvider extends React.Component {
     };
 
     this.mediaQueryListener = this.mediaQueryListener.bind(this);
-  }
 
-  getChildContext() {
-    return this.state;
+    this.mediaRef = this.state.media;
```

Comment:
the name of this `mediaRef` makes it sound like its a react ref when its not

Reply:
You’re right will choose a better name

Reply:
Renamed to : currentMediaState